### PR TITLE
removed udunits2 usages in `shiny/` and `scripts/`

### PR DESCRIPTION
**Pull request by nanu1605**
removed udunits2 from `shiny` and `scripts` folder

### CODE REVIEW

File: shiny/workflowPlot/server_files/model_data_plots_server.R [hunk 1]
```diff
@@ -43,10 +43,10 @@ observeEvent(input$var_name_modeldata, {
 
 # Check that new units are parsible and can be used for conversion
 observeEvent(input$units_modeldata,{
-  if(udunits2::ud.is.parseable(input$units_modeldata)){
+  if(units::install_unit(input$units_modeldata)){
```
Comment (infotroph) [inline shiny/workflowPlot/server_files/model_data_plots_server.R:46]: As we discussed in PEcAn.DB, install_units isn't the right approach here: `ud.is.parseable` checks if the udunits library knows how to handle the unit, `install_unit` declares the unit as valid even if the udunits library has never seen it before. 

What we did in PEcAn.DB was to wrap `as_unit()` in a tryCatch, and that's probably the right approach here too. A shorter but probably much harder to understand hack would be `if( PEcAn.utils::ud_convert(1, input$units_modeldata, input$units_modeldata)) { #test if unit parseable by trying to convert to itself`

File: shiny/workflowPlot/server_files/model_plots_server.R [hunk 2]
```diff
@@ -34,11 +34,11 @@ observeEvent(input$var_name_model, {
 
 # Check that new units are parsible and can be used for conversion
 observeEvent(input$units_model,{
-  parseable <- ifelse(udunits2::ud.is.parseable(input$units_model), "can", "cannot")
-  if(udunits2::ud.is.parseable(input$units_model)){
+  parseable <- ifelse(units::install_unit(input$units_model), "can", "cannot")
+  if(units::install_unit(input$units_model)){
```
Comment (infotroph) [inline shiny/workflowPlot/server_files/model_plots_server.R:38]: same comment as above

**Comment by nanu1605** [comment]
@infotroph I think when PR #2999 will be merged the CI checks will pass

**Review by infotroph — CHANGES_REQUESTED** [review]

File: shiny/workflowPlot/server_files/model_data_plots_server.R (hunk 1 above)
Comment (moki1202) [reply shiny/workflowPlot/server_files/model_data_plots_server.R]: latter is a very cool approach indeed! @nanu1605 please go ahead and try this. If you're confused, ping me or Chris.

Comment (moki1202) [reply shiny/workflowPlot/server_files/model_data_plots_server.R]: I really like @infotroph 's latter suggestion, it's a very cool approach but now that you've written a replacement function for `ud.is.parseable`, go ahead and replace these calls with the `unit_is_parseable` function once you get the green light from other reviewers.

Comment (infotroph) [reply shiny/workflowPlot/server_files/model_data_plots_server.R]: After thinking more, I think my latter suggestion is a bad one -- it's clever but too easy to misunderstand. I can easily see a future reader (including myself in a year!) saying "The input and output units are the same, so this line does nothing! I can delete it!", so defining a new function with a clear name seems preferable.

File: shiny/workflowPlot/server_files/model_plots_server.R [hunk 3]
```diff
@@ -34,11 +34,11 @@ observeEvent(input$var_name_model, {
 
 # Check that new units are parsible and can be used for conversion
 observeEvent(input$units_model,{
-  parseable <- ifelse(udunits2::ud.is.parseable(input$units_model), "can", "cannot")
-  if(udunits2::ud.is.parseable(input$units_model)){
+  parseable <- ifelse(PEcAn.utils::unit_is_parseable(input$units_model), "can", "cannot")
```
Comment (infotroph) [inline shiny/workflowPlot/server_files/model_plots_server.R:37]: This line seems unused, so I think we can delete it:
```suggestion
```

**Review by infotroph — CHANGES_REQUESTED** [review]

File: shiny/workflowPlot/server_files/model_plots_server.R (hunk 3 above)
Comment (infotroph) [reply shiny/workflowPlot/server_files/model_plots_server.R:37]: (I went ahead and committed this so that the final round of checks could kick off)

**Review by infotroph — APPROVED** [review]

**Pull request merged**